x11-proto/xorgproto/xorgproto-2018.4.recipe

```diff
@@ -0,0 +1,53 @@
+SUMMARY="X.Org's protocol headers"
+DESCRIPTION="The X.Org protocol headers."
+HOMEPAGE="https://cgit.freedesktop.org/xorg/proto/xorgproto/"
+COPYRIGHT="1991 Oracle and/or its affiliates. All rights reserved.
+	1985, 1987, 1988, 1990, 1991, 1993-1996, 1998 The Open Group
+	1987 Digital Equipment Corporation, Maynard, Massachusetts.
+	1987 Apollo Computer Inc., Chelmsford, Massachusetts.
+	1989 Hewlett-Packard Company.
+	1999 The XFree86 Project Inc."
+LICENSE="MIT (no promotion)"
+REVISION="1"
+SOURCE_URI="https://xorg.freedesktop.org/archive/individual/proto/xorgproto-$portVersion.tar.bz2"
+CHECKSUM_SHA256="fee885e0512899ea5280c593fdb2735beb1693ad170c22ebcc844470eec415a0"
+PATCHES="xorgproto-$portVersion.patchset"
+
+ARCHITECTURES="any"
+
+PROVIDES="
+	xorgproto = $portVersion
+	devel:xorgproto = $portVersion
```

Comment:
@Begasus 

What I meant is something like this. This way, we don't have to modify other packages, and can easily switch back to separate package, should other contributors dislike the idea :)

```diff
-	devel:xorgproto = $portVersion
+	devel:applewmproto = 1.4.2
+	devel:bigreqsproto = 1.1.2
+	and so on
```
(names and versions are found in `meson.build` file in the tarball)

Reply:
Sounds reasonable, but wouldn't haikuporter pull xproto from the depot atm instead of using the one declared in xorgproto?

Reply:
Unless you have xorgproto installed, that is :)

And I think xorgproto should confilcts with all other protos